src/runtime/micro/micro_common.cc

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file micro_common.cc
+ * \brief common utilties for uTVM
+ */
+
+#include <tvm/runtime/c_runtime_api.h>
+#include <tvm/runtime/registry.h>
+#include <cstdio>
+#include <string>
+#include <sstream>
+#include <cstdint>
+#include "micro_session.h"
+#include "micro_common.h"
+#include "low_level_device.h"
+
+namespace tvm {
+namespace runtime {
+
+DevBaseOffset DevAddr::operator-(DevBaseAddr base) const {
```

Comment:
consider inline those implementations into DevAddr as inline functions. Because they are short, and should be inlined to the caller(rather than become its own function)